README.md

```diff
@@ -27,7 +27,7 @@ Code generation has the following dependencies:
   - The static website generator (hugo)
 
 A few system dependencies are required:
-  - Go >= 1.12
+  - Go >= 1.14
```

Comment:
Should we jump to 1.15?

Reply:
Our SDK generator image that I used for the update is still on 1.14. We probably should update it eventually, but I didn't want to slow down this release for that update.